02 - Community Expansions/02.05 All Community Expansions/Themes/Friendly.md

```diff
@@ -35,7 +35,18 @@ Modes: [[Dark-mode themes|dark]]
 
 ## Features
 
-
+Custom support for:
+
+- ITS-like asides, image captioning and positioning (no callouts because I don't understand them yet);
```

Comment:
Would you be so kind to remove the semicolons and the trailing dot at the end of the list items? Again, just for style/formatting consistency.

Reply:
removed